Resources/Prototypes/Maps/omega.yml

```diff
@@ -2,8 +2,8 @@
   id: Omega
   mapName: 'Omega'
   mapPath: /Maps/omega.yml
-  minPlayers: 10
-  maxPlayers: 50
+  minPlayers: 0
```

Comment:
same as the other lowpop maps adjusted. ame-only map unless your engineer passes the teg knowledge check